src/components/universal/core/Badge/Badge.recipe.ts

```diff
@@ -0,0 +1,40 @@
+import { defineRecipe } from "@pandacss/dev";
+
+export const badgeRecipe = defineRecipe({
+  className: "badge",
+  description: "The styles for the Badge component",
+  base: {
+    borderRadius: "full",
+    fontWeight: "medium",
+    display: "inline-flex",
+    alignItems: "center",
+    userSelect: "none",
+    whiteSpace: "nowrap",
+    textStyle: "sm",
+    px: 2.5,
+    h: 6,
+    gap: 2,
+    background: "accent.default",
+    color: "accent.fg",
+  },
+  variants: {
+    variant: {
+      subtle: {
+        background: "accent.subtle",
+        borderColor: "border.accent",
+        borderWidth: "1px",
```

Comment:
I think we could get rid of the border styles here still for the subtle variant. But that is totally a personal preference thing! This PR is looking great! Going to approve! If you decide you like it better with the border styles, send it my dude!